docs/designers-developers/developers/tutorials/format-api/2-toolbar-button.md

```diff
@@ -0,0 +1,37 @@
+# Add a button to the toolbar
+
+Now that the format is avaible, the next step is to surface it to the UI. You can make use of the [`RichTextToolbarButton`](/packages/editor/src/components/rich-text/README.md#RichTextToolbarButton) component to extend the format toolbar.
+
+Paste this code in `my-custom-format.js`:
+
+```js
+( function( wp ) {
+	var MyCustomButton = function( props ) {
+		return wp.element.createElement(
+			wp.editor.RichTextToolbarButton, {
+				icon: 'editor-code',
+				title: 'Sample output',
+				onClick: function() {
+					console.log( 'toggle format' );
+				},
+			}
+		);
+	}
+	wp.richText.registerFormatType(
+		'my-custom-format/sample-output', {
+			title: 'Sample output',
+			tagName: 'samp',
+			className: null,
+			edit: MyCustomButton,
+		}
+	);
+} )( window.wp );
+```
+
+**Important**: note that this code is using two new utilities (`wp.element.createElement`, and `wp.editor.RichTextToolbarButton`) so don't forget adding the corresponding `wp-element` and `wp-editor` packages to the dependencies array in the PHP file along with the existing `wp-rich-text`.
+
+Let's check that everything is working as expected. Reload the post/page and select a text block. Make sure that the new button was added to the format toolbar, it uses the [editor-code dashicon](https://developer.wordpress.org/resource/dashicons/#editor-code), and the hover text is what you set in the title:
+
+![Toolbar with custom button](/docs/designers-developers/assets/toolbar-with-custom-button.png)
```

Comment:
Does this URL need to point to GitHub raw CDN or is this fine? cc @gziolo do you happen to know?

Reply:
I have no idea, I would give it a go as is and update if necessary. It takes only 15 minutes to see the handbook updated :)
For what's it is worth, it displays correctly on GitHub.

Reply:
This should be okay but we have started uploading graphics to the make/design P2, so we may want to consider updating here too.

Reply:
As per https://wordpress.slack.com/archives/C02RP4WU5/p1547229005149900 it looks like this wouldn't work. Going on to use GitHub CDN while we figure the best strategy going forward.

Reply:
> It takes only 15 minutes to see the handbook updated :)

It actually takes up to half an hour! See https://wordpress.slack.com/archives/C02QB8GMM/p1547058328181200